doc/user_guide/tutorial.md

```diff
@@ -0,0 +1,472 @@
+| 4053 - 40193 Adwar H. Nikola | ADWAR_H_NIKOLA |
+| 5074 - 54002 Ahzmi Inc       | AHZMI_INC      |
+| &hellip;                     | &hellip;       |
+| Yellow Cab                   | YELLOW_CAB     |
+| &hellip;                     | &hellip;       |
+| 3141 - 87803 Zip Cab         | ZIP_CAB        |
+| 3141 - Zip Cab               | ZIP_CAB        |
+| &hellip;                     | &hellip;       |
+| 5129 - 87128                 | `NULL`         |
+
+As you can see, the mapping is not perfect. There are two rows that produce the same ID (`ZIP_CAB`) and one that produces a `NULL`.
+
+In a real-world scenario you would spend more effort to disambiguate the ID generation and make sure that `NULL`s don't happen. For the sake of this tutorial though the naive approach will have to suffice.
+
+## Protecting the Data with RLS
+
+### Creating the RLS Virtual Schema
+
+Exasol's row-level security implementation is a [Virtual Schema](https://github.com/exasol/virtual-schemas). If you know how views work, then consider Virtual Schemas a closely related concept.
+
+A Virtual Schema is a projection of an underlying concrete schema. In the case of RLS it adds a filter layer that makes sure that users only see what they are supposed to.
+
+Please refer to the [user guide](user_guide.md#creating-the-virtual-schema) for detailed instructions on how to install the RLS package.
+
+```sql
+CREATE OR REPLACE JAVA ADAPTER SCRIPT RLS_VSADAPTER_SCHEMA.RLS_VSADAPTER AS
+    %scriptclass com.exasol.adapter.RequestDispatcher;
+    %jar /buckets/bfsdefault/jars/row-level-security-dist-1.0.1.jar;
+/
+;
+
+
+CREATE CONNECTION EXASOL_JDBC_CONNECTION 
+TO 'jdbc:exa:localhost:8563' 
+USER 'BACP' 
+IDENTIFIED BY '<password>';
+
+CREATE VIRTUAL SCHEMA RLS_CHICAGO_TAXI_VS 
+USING RLS_VSADAPTER_SCHEMA.RLS_VSADAPTER 
+WITH
+SQL_DIALECT     = 'EXASOL_RLS'
+CONNECTION_NAME = 'EXASOL_JDBC_CONNECTION'
+SCHEMA_NAME     = 'CHICAGO_TAXI'
+IS_LOCAL        = 'true';
+```
+
+Note that `IS_LOCAL` is important.
+
+### Creating a User Account
+
+Next step is to create a user account that you will need in order to see the effect RLS has.
+
+One of the taxi companies is called "Zip Cab" and the corresponding entry in the `EXA_ROW_TENANT` column is `ZIP_CAB`.
+
+So please create a user account with that name, allow it to log in and grant the `SELECT` privilege on the RLS Virtual Schema.
+
+```sql
+CREATE USER ZIP_CAB IDENTIFIED BY "<password>";
+GRANT CREATE SESSION TO ZIP_CAB;
+GRANT SELECT ON RLS_CHICAGO_TAXI_VS TO ZIP_CAB;
+```
+
+### Querying the Data as Non-privileged User
+
+Now is the moment of truth. You are going to impersonate a taxi company and verify that we only see entries that this company should be able to see.
+
+Log in first with the owner account `BACP` and run this query:
+
+```sql
+SELECT COMPANY, COUNT(1)
+FROM CHICAGO_TAXI.TRIPS
+GROUP BY COMPANY
+ORDER BY COMPANY;
+```
+
+Now login as `YELLOW CAB` and run this query for comparison:
```

Comment:
Is it a user name? We created a `ZIP_CAB` user only

Reply:
Fixed.